webui/react/src/services/api.ts

```diff
@@ -343,6 +343,12 @@ export const timeSeries = generateDetApi<
   Type.TrialSummary[]
 >(Config.timeSeries);
 
+export const getTrialCheckpoints = generateDetApi<
+  Service.getExperimentCheckpointsParams,
+  Api.V1GetTrialCheckpointsResponse,
+  Type.CheckpointPagination
+>(Config.getExperimentCheckpoints);
```

Comment:
this is calling the wrong config
```suggestion
>(Config.getTrialCheckpoints);
```